test/e2e/pivoting.go

```diff
@@ -392,50 +395,30 @@ func RemoveDeployment(ctx context.Context, inputGetter func() RemoveDeploymentIn
 	Expect(err).To(BeNil(), "Failed to delete %s Deployment", deploymentName)
 }
 
-func labelBMOCRDs(targetCluster framework.ClusterProxy) {
-	labels := []string{
-		"clusterctl.cluster.x-k8s.io=",
-		"cluster.x-k8s.io/provider=metal3",
-	}
-	kubectlArgs := ""
-	if targetCluster != nil {
-		kubectlArgs = fmt.Sprintf("--kubeconfig=%s", targetCluster.GetKubeconfigPath())
-	}
-
+func labelBMOCRDs(ctx context.Context, targetCluster framework.ClusterProxy) {
+	labels := map[string]string{}
+	labels[clusterctlv1.ClusterctlLabel] = ""
+	labels[clusterv1.ProviderNameLabel] = "metal3"
 	crdName := "baremetalhosts.metal3.io"
-	for _, label := range labels {
-		var cmd *exec.Cmd
-		if kubectlArgs == "" {
-			cmd = exec.Command("kubectl", "label", "--overwrite", "crds", crdName, label) //#nosec G204:gosec
-		} else {
-			cmd = exec.Command("kubectl", kubectlArgs, "label", "--overwrite", "crds", crdName, label) //#nosec G204:gosec
-		}
-		err := cmd.Run()
-		Expect(err).To(BeNil(), "Cannot label BMO CRDs")
-	}
+	config, err := clientcmd.BuildConfigFromFlags("", targetCluster.GetKubeconfigPath())
```

Comment:
```suggestion
	targetClusterClient := targetCluster.GetClient()
```
Might be easier if that does the same job

Reply:
I have changed it kindly check.